content/recipes/reusability/miappe.md

```diff
@@ -0,0 +1,429 @@
+
+12. Fill out the remaining details by reference to the values to the best of your knowledge. Only fields marked with a ***** are required, but it is best to fill as many fields as possible.
+
+    **Hint:** You can always find out more about what a field means and what information it accepts by hovering over the blue 'i'.
+
+13. By the time you have completed this, your sample will be well-annotated and     understandable to people finding it in the database later. 
+
+    You are ready to submit when all the checks on the right of the page are green ticks, and none are red crosses.
+
+
+    Click '**Submit**'.
+
+14. Now navigate to the 'Samples' tab to see the sample you just registered. You might need to refresh the page.
+
+	Make a note of its accession numbers as you will need these later:
+
+		 ERS####### and SAMEA####### 
+
+
+#### The Read Data
+
+Now that study and sample metadata have been registered, it is time to submit the read data you have produced.
+
+
+
+1. Return to the '**New Submission**' tab in the submission service. You may need to click the '**Restart Submission**' button at the bottom of the page.
+2. Find the '**Submit sequence reads and experiments**' radio button and click
+3. Next, you are asked to choose which study these reads should be added to. Select the study you registered earlier and click '**Next**'.
+4. Now, you have the opportunity to register samples as part of this submission. However, you already registered a sample, so click '**Skip**'.
+5. Now, select '**One Fastq file (Single)**' from the list of file options.
+6. Fill out the form to describe your experiments and register the files. Ordinarily, you would need to upload the files to a staging area before submitting them through this interface, refer to the [ENA upload guidelines](https://ena-docs.readthedocs.io/en/latest/submit/fileprep/upload.html). 
+
+    Appropriate metadata for your read data submission should look similar to this:
+
+
+    * **Sample reference:** *<enter the BioSamples accession here>*
+    * **Instrument Model:** Illumina HiSeq 2500
+    * **Library Name:** barley_library_1
+    * **Library Source:** GENOMIC
+    * **Library Selection:** Restriction Digest
+    * **Library Strategy:** GBS
+    * **Library Layout:** SINGLE
+    * **First File Name:** HOR_1361_BRG.fastq.gz
+
+
+    **Hint**: If you did not note down your sample accession, you can safely visit the ‘**Samples**’ tab, copy the accession, then return to the New Submission tab: your progress will be saved.
+
+7. Click '**Submit**' and see if your submission validates successfully. If you encounter errors, try using the '**Download Template Spreadsheet'** button, open the file and check it in this plain text format; it can be easier to fix errors this way than in the interface. 
+
+
+    Note that the use of template spreadsheets is the best way to submit multiple datasets through this interface.
```

Comment:
````suggestion
```{note}
Note that the use of template spreadsheets is the best way to submit multiple datasets through this interface.
```
````